main.tf

```diff
@@ -1,18 +1,35 @@
+terraform {
+  required_providers {
+    archive = {
+      source = "hashicorp/archive"
+      version = "2.2.0"
```

Comment:
will there be an issue using strict "=" versions? Should we use ">= 2.2.0"? If our wrapper terraform script switches to a newer archive provider, such as 2.3.0, then it would conflict with this version requirement.

Reply:
Having thought about it, I am fine using fixed provider versions until we are able to run this in Terraform 0.14 with the dependency lock feature.